corehq/apps/reports/standard/cases/case_list_explorer.py

```diff
@@ -45,6 +47,17 @@ class CaseListExplorer(CaseListReport):
         SelectOpenCloseFilter,
     ]
 
+    @property
+    @memoized
+    def es_results(self):
+        timer = datadog_bucket_timer(
+            'commcare.case_list_explorer_query.es_timings',
+            tags=['domain:{}'.format(self.domain)],
```

Comment:
It's probably been done in other places, but this is generally not a good practice since there will be many many values of this tag. There is a limit to the number of datadog tags we're allowed to have. The way I've done this in the past is to only tag domains I care about watching:

https://github.com/dimagi/commcare-hq/blob/21a6e0be2b42f256ffb7dff1587e849e85dec25c/corehq/ex-submodules/casexml/apps/phone/restore.py#L711-L712

Reply:
:+1: This feature flag is currently only turned on a for a very small number of projects, but I think just getting a sense of what these timings are like is already a good start. If we find that these timings are higher than we deem acceptable, I can start adding more data about where the high timings are coming from or supplement from the ES slowlogs. 
https://github.com/dimagi/commcare-hq/pull/21395/commits/221c48f4ba8e5839742107701a1b127b2032bbd2